cypress/e2e/marc-authority/edit-marc-authority-save-keep-editing.cy.js

```diff
@@ -0,0 +1,242 @@
+      marcData: {
+        headingTypeFrom1XX: 'Jackson, Peter - edited',
+        headingType: 'Personal name',
+        status: 'Current',
+      },
+    },
+  };
+  const marcFiles = [
+    {
+      marc: 'marcAuthFileForC360092-C360093.mrc',
+      fileName: `testMarcFile.${getRandomPostfix()}.mrc`,
+      numOfRecords: 1,
+    },
+  ];
+  const createdAuthorityIDs = [];
+
+  before('Upload files', () => {
+    cy.loginAsAdmin({ path: TopMenu.dataImportPath, waiter: DataImport.waitLoading }).then(() => {
+      marcFiles.forEach((marcFile) => {
+        DataImport.verifyUploadState();
+        DataImport.uploadFile(marcFile.marc, marcFile.fileName);
+        JobProfiles.waitFileIsUploaded();
+        JobProfiles.waitLoadingList();
+        JobProfiles.search(jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(marcFile.fileName);
+        Logs.checkStatusOfJobProfile('Completed');
+        Logs.openFileDetails(marcFile.fileName);
+        for (let i = 0; i < marcFile.numOfRecords; i++) {
+          Logs.getCreatedItemsID(i).then((link) => {
+            createdAuthorityIDs.push(link.split('/')[5]);
+          });
+        }
+      });
+
+      cy.createTempUser([
+        Permissions.uiMarcAuthoritiesAuthorityRecordEdit.gui,
+        Permissions.uiMarcAuthoritiesAuthorityRecordView.gui,
+        Permissions.uiQuickMarcQuickMarcAuthoritiesEditorAll.gui,
+      ]).then((createdUserProperties) => {
+        testData.userProperties = createdUserProperties;
+        headerContent.editedHeaderContent.source.firstName = testData.userProperties.username;
+
+        cy.login(testData.userProperties.username, testData.userProperties.password);
+      });
+    });
+  });
+
+  beforeEach('Visit MARC Authorities', () => {
+    cy.visit(TopMenu.marcAuthorities);
+    MarcAuthorities.waitLoading();
+    MarcAuthorities.searchBy(testData.authority.searchOption, testData.authority.searchInput);
+    MarcAuthorities.select(`${createdAuthorityIDs[0]}${authorityPostfix}`);
+    MarcAuthority.edit();
+  });
+
+  after('Delete test data', () => {
+    cy.getAdminToken();
+    createdAuthorityIDs.forEach((id) => {
+      MarcAuthority.deleteViaAPI(id);
+    });
+    Users.deleteViaApi(testData.userProperties.userId);
+  });
+
+  it(
+    'C360092 Verify that click on the "Save & keep editing" button doesnt close the editing window of "MARC Authority" record (spitfire) (TaaS)',
+    { tags: [TestTypes.extendedPath, DevTeams.spitfire] },
+    () => {
+      // Verify initial state of edit view
+      QuickMarcEditor.checkHeaderFirstLine(
+        headerContent.initialHeaderContent.marcData,
+        headerContent.initialHeaderContent.source,
+      );
+      QuickMarcEditor.checkButtonsDisabled();
+
+      // Edit record and verify button states updated
+      MarcAuthority.changeField(
+        testData.editedFields[0].tag,
+        `${subfieldPrefix}${testData.editedFields[0].content}`,
+      );
```

Comment:
No assertion from step 3
Entered value is displaying in the "$a" subfield of "1XX" field